src/types.ts

```diff
@@ -32,7 +33,14 @@ type SlotDictionary = {
   [key: string]: Slot
 }
 
-export interface MountingOptions<Props, Data = {}> {
+type RawProps = VNodeProps & {
+  // used to differ from a single VNode object as children
+  __v_isVNode?: never
+  // used to differ from Array children
+  [Symbol.iterator]?: never
+} & Record<string, any>
```

Comment:
This is from vue-next right?

Would it be possible to export it in vue-next? Or at least adding a comment to explain that it is a copy/paste would be great 👍